test/ci/buildkite/run_model_infer.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+# This script is used by BuildKite CI to fetch/run models from a curated model-repo for OV-IE integration project
+WORKDIR=`pwd`
+MODEL=$1
+if [ "${BUILDKITE}" == "true" ]; then
+    echo "--- Model: ${MODEL}"
+fi
+echo "Dir: ${WORKDIR}"
+if [ "${MODEL}" == "" ]; then
+    echo "Error: model not specified!" && exit 1
+fi
+IMAGE=$2
+if [ "${IMAGE}" == "" ]; then
+    echo "Error: image not specified!" && exit 1
+elif [[ ! ${IMAGE} =~ *.* ]]; then
+    IMAGE="${IMAGE}.jpg"
+fi
+INFER_PATTERN=$3
+if [ "${INFER_PATTERN}" == "" ]; then
+    echo "Error: expected pattern not specified!" && exit 1
+fi
+
+REPO=tensorflow_openvino_models_public
+
+if [ "${BUILDKITE}" == "true" ]; then
+    LOCALSTORE_PREFIX=/localdisk/buildkite/artifacts
+else
+    LOCALSTORE_PREFIX=/tmp
+    # invoke locally: .../run_model_infer.sh resnet_50 bike 'mountain bike, all-terrain bike'
```

Comment:
Can you move this comment to the top so the usage of the script is clear when running locally?